rdkit/Chem/Pharm2D/UnitTestLazyGenerator.py

```diff
@@ -12,25 +12,34 @@
 
 """
 import unittest
+
 from rdkit import Chem
-from rdkit.Chem.Pharm2D import SigFactory, LazyGenerator
+from rdkit.Chem.Pharm2D import SigFactory
+
+try:
+  from rdkit.Chem.Pharm2D import LazyGenerator
+except NotImplementedError:
+  LazyGenerator = None
 
 
-class TestCase(unittest.TestCase):
+class TestCase(unittest.TestCase):  # pragma: nocover
 
-  def setUp(self):
-    self.factory = SigFactory.SigFactory()
-    self.factory.SetPatternsFromSmarts(['O', 'N'])
-    self.factory.SetBins([(0, 2), (2, 5), (5, 8)])
-    self.factory.SetMinCount(2)
-    self.factory.SetMaxCount(3)
+  def getFactory(self):
+    factory = SigFactory.SigFactory()
+    factory.SetPatternsFromSmarts(['O', 'N'])
+    factory.SetBins([(0, 2), (2, 5), (5, 8)])
+    factory.SetMinCount(2)
+    factory.SetMaxCount(3)
+    return factory
 
-  def test1(self):
-    """ simple tests
+  def test_NotImplemented(self):
+    assert LazyGenerator is None, 'Review LazyGenerator unit tests'
```

Comment:
Should really be using the UnitTest methods (though the rest of the file doesn't currently do so)